src/test/resources/Logout.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Login
+
+  An a user
+  I want to sign in
+  So I can use the app
+
+  @automated
+  Scenario: User can sign in with valid value
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
```

Comment:
dùng luôn step này cho ngắn:    ` When  user signed in as "John Doe" and "ThisIsNotAPassword"`